client/src/Main.jsx

```diff
@@ -77,6 +96,7 @@ class Main extends React.Component {
   }
 
   render() {
+    console.log('params ', this.props);
```

Comment:
We should remove console.log statements (with the exception of error messages) before pushing to production.